frontend/src/metabase/lib/dom.js

```diff
@@ -513,3 +513,11 @@ export const getEventTarget = event => {
 
   return target;
 };
+
+export function reload() {
+  window.location.reload();
```

Comment:
I think since we now have a utility function, we should refactor all call sites to `window.location.reload()` to `reload()` instead.

e.g. 
- https://github.com/metabase/metabase/blob/f2bf5331bb1d169900efef48ffafa4187896e715/frontend/src/metabase/account/profile/actions.ts#L14
- https://github.com/metabase/metabase/blob/f2bf5331bb1d169900efef48ffafa4187896e715/frontend/src/metabase/account/profile/actions.ts#L14
- https://github.com/metabase/metabase/blob/f2bf5331bb1d169900efef48ffafa4187896e715/frontend/src/metabase/admin/settings/selectors.js#L303


What do you think?

Reply:
I'm more than okay with it, but I made this change only in testing purposes, so if you say we have to replace all the places wtih this utility, I'll do it, but it's definitely not something I wanted to do in scope of this PR

Reply:
I don't think adding a single-line utility function makes sense here.
Everyone knows what `window.location.reload` does.
Having to look up `reload` function and import it results in more code being written and read = more time spent.

What's the point?

Reply:
sounds good on both points.

Reply:
I saw Ryan's comment about mocking `window.location` directly. Would that be better if we can do that then we don't need this util at all?

Reply:
@kamilmielnik @WiNloSt believe me or not, it wasn't my intention to create a useless one-liner just to make this change, unfortunately with latest versions of jsdom they do not allow to override `window` anymore, I tried several ways of mocking and none of them worked. Please check the PR description, especially greyed out block, I tried to explain this change and even provided the link to the code how react-router workarounds it

Reply:
Gotcha. Once you mentioned that I remember I read something about the error that we can't modify `window` no more.

Reply:
https://github.com/jsdom/jsdom/issues/3492
https://github.com/jsdom/jsdom/issues/3492#issuecomment-1537219495

Reply:
thanks for the heads up, I added links to the issues to description to make it even more clear

Reply:
I think I get it now. So this `reload` function won't work in unit tests without mocking it.

IMO `reload` function definitely deserves a comment in code that explains its existence and importance of mocking it in tests.

Reply:
@kamilmielnik yeah, it’s a limitation of the new jsdom, you can’t change implementation of window methods as they are not configurable, that means even if you spy on some window method l, jsdom will fail on location change or on reload as it is not implemented in jsdom (or can’t be implemented, I’m not sure).

good point about the comment, I’ll add it, but I’m not sure where to place it

Reply:
How about as a JSDoc for the `reload` function?

Reply:
added